src/code/z_demo.c

```diff
@@ -1,99 +1,1674 @@
+            break;
+        case 0x14:
+            EnvFlags_Set(globalCtx, 3);
+            break;
+        case 0x15:
+            EnvFlags_Set(globalCtx, 4);
+            break;
+        case 0x16:
+            gSaveContext.playerForm = PLAYER_FORM_DEKU;
+            break;
+        case 0x17:
+            player->stateFlags2 |= 0x4000000;
+            break;
+        case 0x18:
+            player->stateFlags2 &= ~0x4000000;
+            break;
+        case 0x19:
+            sCutsceneStoredPlayerForm = gSaveContext.playerForm;
+            gSaveContext.playerForm = PLAYER_FORM_HUMAN;
+            gSaveContext.equippedMask = PLAYER_MASK_NONE;
+            break;
+        case 0x1A:
+            func_8019F128(NA_SE_EV_EARTHQUAKE_LAST2 - SFX_FLAG);
+            if (isStartFrame) {
+                sCutsceneQuakeIndex = Quake_Add(GET_ACTIVE_CAM(globalCtx), 6);
+                Quake_SetSpeed(sCutsceneQuakeIndex, 30000);
+                Quake_SetQuakeValues(sCutsceneQuakeIndex, 20, 10, 0, 0);
+                Quake_SetCountdown(sCutsceneQuakeIndex, 800);
+            }
+            break;
+        case 0x1B:
+            if (isStartFrame) {
+                globalCtx->nextEntranceIndex = 0x1C00;
+                gSaveContext.nextCutsceneIndex = 0xFFF8;
+                globalCtx->sceneLoadFlag = 0x14;
+                globalCtx->unk_1887F = 3;
+            }
+            break;
+        case 0x1C:
+            if (isStartFrame) {
+                globalCtx->envCtx.unk_17 = 0xD;
+            }
+            break;
+        case 0x1D:
+            gSaveContext.playerForm = sCutsceneStoredPlayerForm;
+            break;
+        case 0x1E:
+            D_801F4DE0 = true;
+            break;
+        case 0x1F:
+            D_801F4DE0 = false;
+            break;
+        case 0x21:
+            if (isStartFrame) {
+                func_80146EE8(globalCtx);
+            }
+            break;
+        case 0x22:
+            if (isStartFrame) {
+                func_80144A94(&globalCtx->sramCtx);
+            }
+            break;
+        case 0x23:
+            if (csCtx->frames != D_801BB15C) {
+                D_801BB15C = csCtx->frames;
+
+                if (REG(15) != 0) {
+                    time = gSaveContext.time;
+                    gSaveContext.time = (u16)REG(15) + time;
+                    time = gSaveContext.time;
+                    gSaveContext.time = (u16)gSaveContext.unk_14 + time;
+                }
+            }
+            break;
+        case 0x24:
+            func_8019F128(NA_SE_EV_EARTHQUAKE_LAST - SFX_FLAG);
+            if (isStartFrame) {
+                sCutsceneQuakeIndex = Quake_Add(GET_ACTIVE_CAM(globalCtx), 6);
+                Quake_SetSpeed(sCutsceneQuakeIndex, 22000);
+                Quake_SetQuakeValues(sCutsceneQuakeIndex, 2, 1, 0, 0);
+                Quake_SetCountdown(sCutsceneQuakeIndex, 800);
+            }
+            break;
+
+        case 0x26:
+            gSaveContext.day = 9;
```

Comment:
I've just remembered, isn't Day 9 the New Day?

Reply:
I though the New Day was day 4

Reply:
Seems like it's literally just the Daytelop that's Day 9.